src/scss/_dashboard.scss

```diff
@@ -1028,6 +1060,7 @@
 
 .uppy-DashboardFileCard[aria-hidden=true] {
   transform: translateY(-105%);
+  transform: translate3d(0, -105%, 0);
```

Comment:
The `translateY` is commented out in some places and kept here, is it intended as a fallback or should they be removed everywhere?

Reply:
Updated, thanks!